QT/mainwindow.cpp

```diff
@@ -39,26 +39,10 @@ void MainWindow::on_FrequencySlider_valueChanged(int value) {
 
 void MainWindow::drawWaveFromFile(QString file)
```

Comment:
Looks like this function is doing nothing. Get rid of it if that's the case.

Reply:
removed